### ISSUE TITLE
Nuker makes entire server unplayable.

### ISSUE THREAD

**Issue by andrewkm** [konsolas/AAC-Issues#1924]
1. Login into Aristois 1.15.2
2. Obtain a shovel, could be unenchanted/stone for all it matters.
3. Use nuker on dirt.
4. Server has a massive lag spike. No kicks for bad packets, even when done for an extended period. (Proof of lag: https://timings.aikar.co/?id=f19ac8ee38864909b556f84096eb5a9d)

Issue is PacketPlayInBlockDig:
https://i.imgur.com/CsqGv27.png

### Core information
**Server version**: 1.15.2 (Paper Build 131)

**AAC version**: 4.3.7

**AAC config**: 4.3.7|1-Default-Celebrimbor

**AAC enabled checks**: All checks enabled.

**ProtocolLib version**: 4.5.1 Build 447

**Plugin list**: _(Plugins on request via Discord - andrewkm-0001)_

### Special environment information

**ViaVersion version**: none

**ProtocolSupport version**: none

**ViaBackwards version**: none

**BungeeCord plugins**: none

**Comment by andrewkm** [comment]
Some more information.
Once the user eventually lags the server out and times out, the following can be seen in console:
https://gist.githubusercontent.com/andrewkm/37e30fd3321b1d06529bf3c55060edcf/raw/d8bcd7a3efc6e6dc9a43a547218a4b45c2edc88b/gistfile1.txt

TL;DR Aristois 1.15.2 nuker not only works but can destroy the server TPS and performance wise.

**Comment by sowhatifimlazy** [comment]
Spottedleaf's Packet Limiter plugin solves this along with some other issues.

**Comment by andrewkm** [comment]
Packet limiter fixes the issue :)
